engine/CSVParser.cc

```diff
@@ -402,15 +403,28 @@ namespace minsky
                       if (c==spec.decSeparator)
                         s+='.';
                       else if (!isspace(c) && c!='.' && c!=',')
-                        s+=c;
+                        s+=c;                    
+                    
+                    // for feature 47, stores all values, duplicates included in a temporary map
+                    auto j=tmpAll.find(key);
+                    try
+                     {
+                      double vv=stod(s);    
+                                          
+                      if (j==tmpAll.end()) tmpAll.emplace(pair<Key,double>(key,vv));  
```

Comment:
I don't understand this code. If you are only inserting data if the key doesn't already exist, then why use a multimap?

Reply:
Yes, that was unnecessary. I've removed it. So is the use of tmpAll. tmpData contains the data that we want to import.